ckanext/harvest/templates/snippets/source_item.html

```diff
@@ -30,9 +30,9 @@ <h3 class="dataset-heading">
         {{ h.link_to(h.truncate(title, truncate_title), url) }}
       {% endif %}
       {% if source.get(state, '').startswith('draft') %}
-        <span class="label label-info">{{ _('Draft') }}</span>
+        <span class="badge bg-info">{{ _('Draft') }}</span>
       {% elif source.get(state, '').startswith('deleted') %}
-        <span class="label label-important">{{ _('Deleted') }}</span>
+        <span class="badge bg-important">{{ _('Deleted') }}</span>
```

Comment:
```suggestion
        <span class="badge bg-danger">{{ _('Deleted') }}</span>
```
The class `label-important` has been marked as deprecated and replaced by `label-danger` which is now `bg-danger`.